app/models/storytime/role.rb

```diff
@@ -6,6 +6,10 @@ class Role < ActiveRecord::Base
 
     validates :name, uniqueness: true
 
+    def admin?
+      name == "admin"
+    end
+
```

Comment:
I should have said this to start, but we should probably have these for all of our roles (editor, writer, etc.)